META6.json

```diff
@@ -8,8 +8,10 @@
     ],
     "license"     : "AGPL-3.0-or-later",
     "provides"    : {
-        "Blin::Module"     : "lib/Blin/Module.pm6",
-        "Blin::Processing" : "lib/Blin/Processing.pm6"
+        "Blin::Module"        : "lib/Blin/Module.rakumod",
+        "Blin::Processing"    : "lib/Blin/Processing.rakumod",
+        "Blin::Tester::Zef"   : "lib/Blin/Tester/Zef.rakumod",
+        "Blin::Tester::Pakku" : "lib/Blin/Tester/Pakku.rakumod"
     },
     "depends"     : [
```

Comment:
I think Pakku should be added as a dependency, right?

Reply:
yes it should, I removed it temporarily because Pakku was [removed](https://github.com/Raku/ecosystem/pull/543/commits/057248a2b9e638409329b5dbdc1b3bb08b978df6) by mistake from the Ecosystem, so I removed it from list of dependencies temporarily untill I get the time to add it back. I added Pakku now to `META6.json` file, but the build will fail.

Reply:
I just created a [PR](https://github.com/Raku/ecosystem/pull/556) to re-add `Pakku` to the Ecosystem.